.zenodo.json

```
@@ -0,0 +1,162 @@
+{
+    "creators": [
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Vay, Jean-Luc"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Almgren, Ann"
+      },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Amorim, L\u00edgia Diana"
+      },
+      {
+        "affiliation": "Laboratoire d'Optique Appliqu\u00e9e",
+        "name": "Andriyash, Igor"
```

@hightower8083 what is your ORCiD? :)
```suggestion
        "name": "Andriyash, Igor",
       "orcid": "..."
```

---

Thanx for including me to this honorable list =) ! 
Shamefully, I've barely ever used ORCiD, but  here is one: 0000-0003-0313-4496